frontend/src/lib/components/LemonRow/LemonRow.scss

```diff
@@ -105,7 +105,24 @@
     padding: 0;
 }
 
-.LemonRow--compact {
+.LemonRow--large {
+    min-height: 3.5rem;
+    padding: 0.5rem 1rem;
```

Comment:
I made a tiny change: made this more in line with the design (where it was 56px tall instead of 60px).